go.mod

```diff
@@ -64,7 +64,7 @@ require (
 	github.com/go-openapi/validate v0.18.0
 	github.com/go-ozzo/ozzo-validation v3.5.0+incompatible // indirect
 	github.com/godbus/dbus v4.1.0+incompatible
-	github.com/gogo/protobuf v0.0.0-20171007142547-342cbe0a0415
+	github.com/gogo/protobuf v1.1.1
```

Comment:
https://github.com/kubernetes/kubernetes/pull/79574 is also in flight and updates this to v1.2.0

I don't have the expertise to know if there are changes beyond the generated .pb.go files that we want or don't want